Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -209,6 +215,76 @@ private void OnTemperatureChangeAttempt(EntityUid uid, TemperatureProtectionComp
         {
             args.Args.TemperatureDelta *= component.Coefficient;
         }
+
+        private void OnParentChange(EntityUid uid, TemperatureComponent component,
+            ref EntParentChangedMessage args)
+        {
+            RecursiveThresholdUpdate(uid);
+        }
```

Comment:
Isn't it possible to short-circuit this? If a mob changes parents which is frequent this is gonna go through like 40 children unnecessarily.